app/templates/CreditCardPayment.html.twig

```diff
@@ -0,0 +1,244 @@
+<div style="height: 42em;">
+	<section class="container clearfix" id="donation-amount">
+		<div class="ltcol">
+			&nbsp;
+		</div>
+
+		<div class="ctcol">
+			<div class="box rounded">
+				<div class="box-footer clearfix">
+					<span class="icon-ok-sign">
+						{$ 'paytext_cc'|trans({}, 'paymentIntervals') $} {$ donation.interval|trans({}, 'paymentIntervals') $} {$ donation.amount|localizedcurrency('EUR', 'de_DE') $}</strong>
+						per Kreditkarte.</span>
+				</div>
+			</div>
+		</div>
+
+		<div class="rtcol">
+			<div class="icon-bar-chart _icon-list-item">
+				<p>
+					<a class="wlightbox inline" href="#" data-href="#wlightbox-wohin-geht-das-geld">
+						<strong>Wohin geht das Geld?</strong>
+					</a>
+				</p>
+			</div>
+
+			<div class="icon-hand-right _icon-list-item" style="margin-top: -9px;">
+				<p>
+					<a class="wlightbox inline" href="#" data-href="#wlightbox-steuerlich-absetzbar">
+						<strong>Steuerlich absetzbar</strong>
+					</a>
+				</p>
+			</div>
+		</div>
+	</section>
+	<div class="steps-overview container clearfix">
+		<div class="ltcol">
+			&nbsp;
+		</div>
+		<div class="ctcol">
+			<div class="box rounded partial no-margin">
+				<div class="box-header">
+					<ul class="step-list horizontal clearfix">
+						<li class="first">Mein Betrag</li>
+						<li class="active">Meine Daten</li>
+						<li class="last">Fertig!</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+		<div class="rtcol">
+			&nbsp;
+		</div>
+	</div>
+	<div>
+		<div class="ltcol">
+			&nbsp;
+		</div>
+
+		<div class="ctcol">
+			<iframe id="micropayment-portal" class="payment-portal" scrolling="auto"
+					style="width:39em; height: 39em; border: 0px;"
+					allowtransparency="true" src="{$ iframeUrl|raw $}">
+			</iframe>
+		</div>
+		<div class="rtcol">
+			<div class="_icon-list-item first">
+				<span class="logo-creditcards"></span>
+			</div>
+			<div class="icon-lock _icon-list-item">
+				<p>
+					<strong>SSL-Verschlüsselte Daten</strong>
+				</p>
+
+				<p>
+					Ihre Spende wird über eine sichere SSL-Verschlüsselung aufgenommen und liegt später auf deutschen
+					Servern.
+				</p>
+			</div>
+			<div class="icon-shield _icon-list-item last">
+				<p>
+					<strong>Ihre Daten sind sicher</strong>
+				</p>
+				<p>
+					Wir fühlen uns dem Datenschutz verpflichtet. Unsere Bestimmungen dazu können Sie
+					<a class="wlightbox inline" href="Datenschutz" data-href="#wlightbox-datenschutz">hier</a> einsehen
+				</p>
+			</div>
+		</div>
+	</div>
+	<div class="wlightbox-contents">
```

Comment:
Please add TODO comment here to put the lightbox contents into separate templates.